GetIntoTeachingApi/Models/CandidatePastTeachingPosition.cs

```diff
@@ -7,7 +7,7 @@
 namespace GetIntoTeachingApi.Models
 {
     [Entity("dfe_candidatepastteachingposition")]
-    public class CandidatePastTeachingPosition : BaseModel
+    public class CandidatePastTeachingPosition : BaseModel, IHasCandidateId

```

Comment:
Could the interface have a more descriptive name like `ICandidateModelUpsertable`? or would that not make sense?

Reply:
Yeah I struggled with this interface name 😞  I settled on `IHasCandidateId` because it was the clearest/most accurate I could come up with; `ICandidateModelUpsertable` suggests if you implement the interface then your record will be upsertable, but that's not the case (it also needs to extend `BaseModel` and have appropriate data annotations)